arcgis-ios-sdk-samples/Scenes/Create terrain surface from a local tile package/README.md

```diff
@@ -0,0 +1,51 @@
+# Create terrain from a local tile package
+
+Set the terrain surface with elevation described by a local tile package.
+
+![Create terrain from a local tile package](create-terrain-from-a-local-tile-package.png)
+
+## Use case
+
+The terrain surface is what the basemap, operational layers, and graphics are draped on. The tile package must be a LERC (limited error raster compression) encoded TPK. Details on creating these are in the [ArcGIS Pro documentation](https://pro.arcgis.com/en/pro-app/help/sharing/overview/tile-package.htm).
+
+Terrain can be loaded offline from tile packages (.tpk).
+
+## How it works
+
+1. Create an `ArcGISScene` and add it to a `SceneView`.
+1. Create an `ArcGISTiledElevationSource` with the path to the local tile package.
+1. Add this source to the scene's base surface: ```
+surface.elevationSources.append(elevationSource)
+scene.baseSurface = surface```.
+
+## Relevant API
```

Comment:
Just wanted to confirm if the formatting/rendering of this heading `## Relevant API` and other ones below are correct? I ask because the one above `## How it works` appears to be in bold, and the rest are not.

Reply:
I retyped these headers so now all H2 tags appear the same on my end. Please let me know if it is not the same for you.

Reply:
`## How it works` appears bold while the other H2 tags don't. If they look ok in the sample, it's fine